data/net/minecraft/world/item/enchantment/Enchantment.mapping

```diff
@@ -1,55 +1,103 @@
 CLASS net/minecraft/world/item/enchantment/Enchantment
+	FIELD category Lnet/minecraft/world/item/enchantment/EnchantmentCategory;
+		COMMENT The category of the enchantment. This often controls which items the enchantment can be applied to and which creative tabs the enchantment will be displayed under.
+	FIELD descriptionId Ljava/lang/String;
+		COMMENT The descriptionId for the enchantment. Commonly used to create localization keys.
+	FIELD rarity Lnet/minecraft/world/item/enchantment/Enchantment$Rarity;
+		COMMENT The rarity of the enchantment.
+	FIELD slots [Lnet/minecraft/world/entity/EquipmentSlot;
+		COMMENT The valid equipment slots for the enchantment.
+	METHOD <init> (Lnet/minecraft/world/item/enchantment/Enchantment$Rarity;Lnet/minecraft/world/item/enchantment/EnchantmentCategory;[Lnet/minecraft/world/entity/EquipmentSlot;)V
+		ARG 1 rarity
+		ARG 2 category
+		ARG 3 slots
 	METHOD byId (I)Lnet/minecraft/world/item/enchantment/Enchantment;
-		COMMENT Gets an Enchantment from the registry, based on a numeric ID.
+		COMMENT Gets an enchantment from the registry using it's internal numeric Id.
 		ARG 0 id
+			COMMENT The internal numeric id of the enchantment.
 	METHOD canEnchant (Lnet/minecraft/world/item/ItemStack;)Z
-		COMMENT Determines if this enchantment can be applied to a specific ItemStack.
+		COMMENT Checks if an enchantment can be applied to a given ItemStack.
```

Comment:
`this` was better, imho. `an` enchantment is very unspecific and could mean any, when in fact it is this one.